web/openshift/templates/config.yaml

```diff
@@ -57,6 +57,14 @@ objects:
 
         # This properly deals with react routes.
         try_files {path} {path}/ /index.html
+        
+        # The site js file caching age, we can add more filetype in 
+        # static object if needed.
+        @static {
+            file
+            path *.js 
+        }
+        header @static Cache-Control max-age=864000
```

Comment:
very close but you are missing cache control for the index.html file

https://create-react-app.dev/docs/production-build/#static-file-caching